unstructured/ingest/connector/google_drive.py

```diff
@@ -103,7 +114,9 @@ def get_file(self):
         from googleapiclient.errors import HttpError
         from googleapiclient.http import MediaIoBaseDownload
 
-        self.config.service = create_service_account_object(self.config.service_account_key)
+        if self.session_handle is None:
+            raise ValueError("Google Drive session handle was not set.")
+        self.session_handle = cast(GoogleDriveSessionHandle, self.session_handle)
```

Comment:
Since this is already set above: 
```python
session_handle: Optional[GoogleDriveSessionHandle] = None
```
I don't think the cast is necessary.

Reply:
iirce linting gets mad if you don't explicitly cast since it was optional